diff --git a/framecpp/test/test_CheckSum.cc b/framecpp/test/test_CheckSum.cc
index a42a7a7aa..35b9b9afb 100644
--- a/framecpp/test/test_CheckSum.cc
+++ b/framecpp/test/test_CheckSum.cc
@@ -251,7 +251,7 @@ BOOST_AUTO_TEST_CASE( test_checksum_8589934592_bytes )
     // thsn 2^32 bytes in length.
 
     static const auto expected_value( expected_value_zero_8589934592 );
-    static seconds    expected_maximum_time( 15 );
+    static seconds    expected_maximum_time( 60 );
 
     cksum_boost test_cksum_boost;
     cksum_ldas  test_cksum_ldas;
